app/models/energy/schedulebase.py

```diff
@@ -43,3 +25,28 @@ class ScheduleUnitType (str, Enum):
     percent = 'Percent'
     control = 'Control'
     mode = 'Mode'
+
+
+class ScheduleType(BaseModel):
+    """Specifies the data types and limits for values contained in schedules."""
+
+    type: Enum('ScheduleType', {'type': 'ScheduleType'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    lower_limit_value: float = Schema(
+        0,
```

Comment:
I think the default here should be `None`, meaning that there is no lower limit and that an empty string ultimately gets written into the IDF.